packages/postcss-lowercase-text/src/__tests__/index.test.js

```diff
@@ -0,0 +1,156 @@
+    'should safely transform the  CSS caps multiple HTML Selectors to lowercase',
+    'UL LI{border: 1px solid black;}',
+    'ul li{border: 1px solid black;}'
+  );
+
+  run(
+    'should safely transform the  CSS HTML Selectors to lowercase',
+    'UL LI, p A{border: 1px solid black;}',
+    'ul li, p a{border: 1px solid black;}'
+  );
+
+  run(
+    'should safely transform the  CSS attribute Selectors to lowercase',
+    '[NAME="newname"]{border: 1px solid black;}',
+    '[name="newname"]{border: 1px solid black;}'
+  );
+
+  run(
+    'should safely transform the  CSS attribute Selectors to lowercase',
+    'DIV#antipattern:NTH-child(3).HORSEHAIR [ID="ding"] { color: yellow; }',
+    'div#antipattern:nth-child(3).HORSEHAIR [id="ding"] { color: yellow; }'
+  );
+
+  run(
+    'should safely transform the  CSS pseudo class Selectors to lowercase',
+    'a:FIRST-CHILD{border: 1px solid black;}',
+    'a:first-child{border: 1px solid black;}'
+  );
+
+  run(
+    'should not transform the  CSS :active pseudo class Selectors to lowercase',
+    'a:ACTIVE{border: 1px solid black;}',
+    'a:ACTIVE{border: 1px solid black;}'
+  );
+
+  run(
+    'should safely transform the  CSS HTML Selectors followed by id to lowercase',
+    'LI#idname{border: 1px solid black;}',
+    'li#idname{border: 1px solid black;}'
+  );
+  run(
+    'should safely transform the  CSS HTML Selectors followed by class to lowercase',
+    'LI.classname{border: 1px solid black;}',
+    'li.classname{border: 1px solid black;}'
+  );
+
+  run(
+    'should safely transform the  CSS nested classname  Selectors to lowercase',
+    'classname{border: 1px solid black;}',
+    'classname{border: 1px solid black;}'
+  );
+});
+describe('transforming CSS units', () => {
+  run(
+    'should safely transform the absolute units to lowercase : px',
+    'classname{border: 1PX solid black;}',
+    'classname{border: 1px solid black;}'
+  );
+
+  run(
+    'should safely transform the absolute units to lowercase : em',
+    '#idname{padding: 1EM 2EM;}',
+    '#idname{padding: 1em 2em;}'
+  );
+
+  run(
+    'should safely transform the angle units to lowercase : deg',
+    '#idname{transform : rotate(20DEG);}',
+    '#idname{transform : rotate(20deg);}'
+  );
+
+  run(
+    'should safely transform the Frequency units to lowercase of ',
+    'p.low { pitch: 105HZ; }',
+    'p.low { pitch: 105hz; }'
+  );
+
+  run(
+    'should safely transform Property and Value ',
+    '.classname { COLOR: RED; }',
+    '.classname { color: red; }'
+  );
+  run(
+    'should safely transform Property ',
+    ':root {  --main-BG-color: coral;} .classname {   color : RED;  background-color: var(--main-BG-color);}',
+    ':root {  --main-BG-color: coral;} .classname {   color : red;  background-color: var(--main-BG-color);}'
+  );
+  run(
+    'should not transform css variable declarations',
+    ':root {  --BG-COLOR: coral; --TEXT-COLOR: RED;}',
+    ':root {  --BG-COLOR: coral; --TEXT-COLOR: red;}'
```

Comment:
Value of variables should not be lowercased

Reply:
they are css values, they can be transformed as well just like the rest of the values.

Like this 

https://github.com/cssnano/cssnano/pull/867/files#diff-fe88a57283a0a1a5c792a3000e5badbaR43-R53

Reply:
I will read spec, maybe you are right